diff --git a/python/libsvm/svm.py b/python/libsvm/svm.py
index 28507e1..0027486 100644
--- a/python/libsvm/svm.py
+++ b/python/libsvm/svm.py
@@ -138,7 +138,15 @@ try:
     from numba import jit
     jit_enabled = True
 except:
-    jit = lambda x: x
+    # We need to support two cases: when jit is called with no arguments, and when jit is called with
+    # a keyword argument.
+    def jit(func=None, *args, **kwargs):
+        if func is None:
+            # This handles the case where jit is used with parentheses: @jit(nopython=True)
+            return lambda x: x
+        else:
+            # This handles the case where jit is used without parentheses: @jit
+            return func
     jit_enabled = False
 
 @jit(nopython=True)
